app/repositories/api_connection.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module ApiConnection
+  class << self
+    def create_default_connection(base_url, adapter: nil, retryable: true)
+      ::Faraday.new base_url do |faraday|
+        faraday.use :finc_repository_exception_connection
+        faraday.request :retry, retry_options if retryable
+        faraday.request :json
+        faraday.response :json, content_type: /\bjson\z/, parser_options: { symbolize_names: true }
+        faraday.response :finc_repository_exception_response
+        faraday.headers['X-Application-Name'] = ::OemNttdataHome::Application.config.application_name
+        faraday.headers['Accept'] = 'application/json'
+        faraday.options.timeout = 12
+        faraday.options.open_timeout = 6
```

Comment:
タイムアウトもっと短くとって良いと思います

```suggestion
        faraday.options.timeout = 3
        faraday.options.open_timeout = 3
```